contractSizeReport.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+const fs = require('fs');
+const util = require('util');
+const got = require('got');
+const yargs = require('yargs');
+
+let path = 'artifacts';
+
+const readdir = util.promisify(fs.readdir);
+
+let argv = yargs.default('branch', 'Katalyst').alias('b', 'branch').argv;
```

Comment:
this is argv input when we run
for example: node contractSizeReport.js -b master
then this will be master. When we change our base branch should change this
PULL_REQUEST_BRANCH is on travis only

Reply:
so, So default branch will be Katalyst for now
was hoping default could be PR branch.

Reply:
default when we run node node contractSizeReport.js is Katalyst but on PR it's PR branch
it's on .travis/report.sh